app/models/match_log_info.rb

```diff
@@ -4,36 +4,41 @@ class MatchLogInfo < ActiveRecord::Base
 
   def get_owner
     source = self.match_source
-    if self.is_player_log? then
+    if self.is_player_log?
       return source.player.user
     end
-    if self.is_referee_log? then
+    if self.is_referee_log?
       return source.manager.referee.user
     end
-    return nil
+    nil
   end
 
   def is_player_log?
-    return self.match_source_type.to_s == "PlayerMatch"
+    self.match_source_type.to_s == "PlayerMatch"
   end
 
   def is_referee_log?
-    return self.match_source_type.to_s == "Match"
+    self.match_source_type.to_s == "Match"
   end
 
   def has_match_source?
-    return self.match_source != nil
+    self.match_source != nil
   end
 
   def has_match_source_with_self?
-    return self.match_source.match_log_info == self
+    self.match_source.match_log_info == self
   end
 
   def has_logs?
-    if self.log_stdout == nil then return false end
-    if self.log_stderr == nil then return false end
-    if self.log_stdout.length == 0 then return false end
-    if self.log_stderr.length == 0 then return false end
-    return true
+    if self.log_stdout == nil
+      return false
+    elsif self.log_stderr == nil
+      return false
+    elsif self.log_stdout.length == 0
+      return false
+    elsif self.log_stderr.length == 0
+      return false
+    end
+    true
   end
```

Comment:
These lines can be shorted to
```
!self.log_stdout.nil? && self.log_stdout.length > 0 && !self.log_stderr.nil? && self.log_stderr.length > 0
```
Even better is to make some helper function like
```
def valid_log?(log)
  !log.nil? && log.length > 0
end
```
and replacing the code with
```
valid_log?(self.log_stdout) && valid_log?(self.log_stderr)
```